Formula/core-lightning.rb

```diff
@@ -0,0 +1,41 @@
+class CoreLightning < Formula
+  desc "Lightning Network implementation focusing on spec compliance and performance"
+  homepage "https://github.com/ElementsProject/lightning"
+  url "https://github.com/ElementsProject/lightning.git",
+      using:    :git,
+      tag:      "v23.05",
+      revision: "d1cf88c62e8ff10485f3b40cddb93fc0063ba92a"
+  license "MIT"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gettext" => :build
+  depends_on "gmp" => :build
+  depends_on "gnu-sed" => :build
+  depends_on "libsodium" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "poetry" => :build
+  depends_on "postgresql@14" => :build
+  depends_on "protobuf" => :build
+  depends_on "python@3.8" => :build
+  depends_on "bitcoin"
+
+  def install
+    File.open("external/lowdown/configure.local", "a") do |configure_local|
+      configure_local.puts "HAVE_SANDBOX_INIT=0"
+    end
+    system "poetry", "env", "use", "3.8"
+    system "poetry", "install"
```

Comment:
Where does this install the required Python libraries?

Reply:
Poetry manages a virtualenv directory that contains the libraries.

Reply:
Where does it put the virtualenv directory?

Reply:
`~/Library/Caches/pypoetry/virtualenvs`

Reply:
Are these libraries needed at run-time?

Reply:
Not to my knowledge. This is a C app.